workspaces/analytics/.changeset/famous-eggs-notice.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage-community/plugin-analytics-module-ga4': minor
+---
+
+Allow defining custom cookie flags
```

Comment:
probably we should call out that we are setting a new default flag that silence the warning.

Reply:
better now?